docs/understand/roadmap.md

```diff
@@ -156,68 +166,21 @@ This release will mark the completion of project **["lakeFS on the Rocks"](https
 
 [Track and discuss on GitHub](https://github.com/treeverse/lakeFS/blob/master/design/open/metadata_kv/index.md){: target="_blank" class="btn" }
 
+### Ref-store implementation for DynamoDB
```

Comment:
Really looking forward to it :)
A couple of things that I think are worth taking into account/validate:

- DynamoDB costs can be high if it's misused (i.e not utilizing the 1K WCU)
- Performance - will we get the same commit guarantees we have with Postgres today?

Reply:
That's probably correct - but is on-par with other tools in the space (namely, Delta 1.2 that was just released, EMRFS, etc), that standardize on DynamoDB for similar purposes